layers/newrelic/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG PHP_VERSION
+ARG NEW_RELIC_LICENSE_KEY
+ARG NEW_RELIC_DAEMON_ADDRESS
+FROM bref/build-php-$PHP_VERSION AS ext
+
+# Build the New Relic Agent install silently for PHP Agent 9.14
+# Then append php.ini with key information including a license key and 
+# location of the daemon coming from an ENV VAR for Lambda
+
+RUN \
+  curl -L http://download.newrelic.com/php_agent/release/newrelic-php5-9.14.0.290-linux.tar.gz | tar -C /tmp -zx && \
+  export NR_INSTALL_USE_CP_NOT_LN=1 && \
+  export NR_INSTALL_SILENT=1 && \
+  /tmp/newrelic-php5-*/newrelic-install install && \
+  rm -rf /tmp/newrelic-php5-* /tmp/nrinstall* 
+  
+RUN echo $' \n \
+extension=newrelic.so \n \
+newrelic.appname = ${NEW_RELIC_APP_NAME} \n \
+newrelic.license = ${NEW_RELIC_LICENSE_KEY} \n \
+newrelic.daemon.address = ${NEW_RELIC_DAEMON_ADDRESS} \n \
+newrelic.logfile = ${NEW_RELIC_LOG_FILE} \n \
+newrelic.loglevel = ${NEW_RELIC_LOG_LEVEL} \n \
+newrelic.daemon.app_connect_timeout = ${NEW_RELIC_DAEMON_APP_CONNECT_TIMEOUT} \n \
+newrelic.daemon.start_timeout = ${NEW_RELIC_DAEMON_START_TIMEOUT} \n \
+' >> /opt/bref/etc/php/php.ini
+
+# newrelic.so will be installed to `php-config --extension-dir`/newrelic.so
+# In this core image newrelic.ini will be placed but not exist in the scatch layer
+# ${INSTALL_DIR}/etc/php/php.ini /tmp/php.ini - php.ini will contain the needed config and extension ref
+
+# Copy the newrelic.so and php.ini file to the tmp folder
+
+RUN cp `php-config --extension-dir`/newrelic.so /tmp/newrelic.so
+RUN cp /opt/bref/etc/php/php.ini /tmp/php.ini
```

Comment:
```suggestion
RUN echo 'extension=/opt/bref-extra/newrelic.so' > /tmp/ext.ini
```

Reply:
I'm confused by this. php.ini is where the .so is loaded. What is ext.ini in this context and the extension= is already achieved in the Dockerfile inside the php.ini with the configuration settings.